grails-app/services/au/org/ala/biocollect/merit/RoleService.groovy

```diff
@@ -28,14 +28,13 @@ class RoleService {
     }
 
     public List getRoles() {
-        def allRoles = roles() // cached
-
+        List<String> allRoles = roles() // cached
         if (allRoles.size() <= 1) {
             // possible empty list or value, due to previous WS call timing out
             allRoles = roles(true) // reload with cleared cache
         }
 
-        return allRoles
+        allRoles.findAll { it == PROJECT_ADMIN_ROLE || it == PROJECT_EDITOR_ROLE }
```

Comment:
There is also a project participant role, which is created when the survey allows public access.